### PR TITLE
Misc. typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 --skip="./lang"`

### CODE REVIEW

File: core/classes/DbQuery.class.php [hunk 1]
```diff
@@ -196,7 +196,7 @@ public function append_sql( $p_query_string ) {
 	}
 
 	/**
-	 * Creates a string containing a parameter that can be appended to que query string
+	 * Creates a string containing a parameter that can be appended to queue query string
```
Comment (gheeshuggah) [inline core/classes/DbQuery.class.php]: `s/que/queue/` ?

File: core/classes/DbQuery.class.php [hunk 2]
```diff
@@ -51,7 +51,7 @@
  *     - "${string}{int}" For special constructs, eg: $in0 for late binding IN caluses
  * 2) $expanded_query_string: stores the query string after expansion of special constructs
  *   into standard "${int}" parameters
- * 3) $db_query_string: stores que query string after all paramaters have been renamed,
+ * 3) $db_query_string: stores queue query string after all parameters have been renamed,
```
[the remark this answers is not in the document]
Comment (dregad) [reply core/classes/DbQuery.class.php]: @cproensa did you mean 'stores _the_ query string' ?

File: core/classes/DbQuery.class.php (hunk 1 above)
Comment (dregad) [reply core/classes/DbQuery.class.php]: @cproensa same as above

File: core/classes/DbQuery.class.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (cproensa) [reply core/classes/DbQuery.class.php]: > did you mean 'stores the query string' ?

Yes

File: core/classes/DbQuery.class.php (hunk 1 above)
Comment (cproensa) [reply core/classes/DbQuery.class.php]: > did you mean 'stores the query string' ?

Yes

File: core/classes/DbQuery.class.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (dregad) [reply core/classes/DbQuery.class.php]: Please fix

**Comment by luzpaz** [comment]
fixed and re-pushed

File: core/classes/DbQuery.class.php (hunk 1 above)
Comment (dregad) [reply core/classes/DbQuery.class.php]: Please fix

File: javascript_translations.php [hunk 3]
```diff
@@ -15,7 +15,7 @@
 # along with MantisBT.  If not, see <http://www.gnu.org/licenses/>.
 
 /**
- * Handle langauge translations for Javascript
+ * Handle language translations for Javascript
```
Comment (dregad) [inline javascript_translations.php]: Remove this change, there is no point in patching comments in a 3rd party library

File: library/rssbuilder/doc/changelog_rssbuilder.htm [hunk 4]
```diff
@@ -26,7 +26,7 @@ <h3>2.2 <small>(2005-12-04)</small></h3>
 <h3>2.1 <small>(2004-03-25)</small></h3>
 <ul>
 	<li>Updated code for PHP&nbsp;5 RC1. Will not work under PHP&nbsp;4!</li>
-	<li>Completly rewritten classes which take use of PHP5's new DOM functions!</li>
+	<li>Completely rewritten classes which take use of PHP5's new DOM functions!</li>
```
Comment (dregad) [inline library/rssbuilder/doc/changelog_rssbuilder.htm]: Remove this change, there is no point in patching comments in a 3rd party library

File: library/rssbuilder/rss_sample_script.php [hunk 5]
```diff
@@ -21,7 +21,7 @@
 ob_start();
 include_once 'class.RSSBuilder.inc.php';
 
-/* create the object - remember, not all attibutes are supported by every rss version. just hand over an empty string if you don't need a specific attribute */
+/* create the object - remember, not all attributes are supported by every rss version. just hand over an empty string if you don't need a specific attribute */
```
Comment (dregad) [inline library/rssbuilder/rss_sample_script.php]: Remove this change, there is no point in patching comments in a 3rd party library

**Comment by vboctor** [comment]
Merged via 724dde4f6338e574dde5ad21de4858740e1075b2.  Thanks @luzpaz

**Pull request closed without merging**